vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -76,6 +89,11 @@ cut_official_branch() {
     git add .
     git commit -m"chore: remove release notes in official branch"
     
+    # Compute and store the VHD build timestamp for hotfixes
+    find_and_write_build_timestamp
+    git add .
+    git commit -m"chore: compute and store VHD build timestamp in official branch"
```

Comment:
nit there should probably be a space after the -m

Reply:
ah yes, done